graphdatascience/graph/graph_object.py

```diff
@@ -28,6 +28,12 @@ def _graph_info(self, yields: List[str] = []) -> Series:
 
         return info.squeeze()  # type: ignore
 
+    def database(self) -> str:
+        return self._graph_info(["database"])  # type: ignore
```

Comment:
it would be really neat if we had the feature to control `YIELD` given this kind of API. At the moment we `YIELD *` and then pick on the client side what to actually use, right? Which means that we end up computing the expensive `degreeDistribution` for things like this. (Fortunately it is cached, so we only pay once, but still.)

Reply:
Oh we have that feature already? That's great.